scripts/hid_configurator/NrfHidDevice.py

```diff
@@ -211,7 +213,18 @@ def open_devices(vid):
                         dir_devs[hwid] = discovered_dev
                         dev_active = True
 
-                peers = NrfHidDevice._get_connected_peers(dev, LOCAL_RECIPIENT)
+                peers, peers_cache = NrfHidDevice._get_connected_peers(dev, LOCAL_RECIPIENT)
```

Comment:
If you separate functions, you can poll peer cache here and then after handling it you can poll for connected peers

Reply:
I would prefer to get peers cache after indexing peers to prevent accidentally triggering assertion failure (the one in config_event logging implementation) on peripherals with older firmware.

Reply:
Also, as stated in previous comment, later on we could use peers cache only internally in the function.